tests/unit/pretender-query-param-handler-test.js

```diff
@@ -90,6 +90,33 @@ module('pretender-query-params-handler', function () {
       );
     });
 
+    test('can match a request with query params in any order', async function (assert) {
+      this.server = new QueryParamAwarePretender({
+        normalizeURLs: true,
+      });
```

Comment:
specifying `normalizeURLs` doesn't do much here since the PR has the default as `true`

but I think this test is actually good and we should make the default false, i.e. have the default implementation be "dumber" but with options for making things more clever where users know it's safe for their application (such as ignoring param order).

Reply:
Agreed! Defaulted to `false`